web/src/main/java/org/cbioportal/web/StudyViewController.java

```diff
@@ -196,147 +187,15 @@ private void init() {
     )
     public List<ClinicalDataBin> cachableFetchClinicalDataBinCounts(
```

Comment:
You need an annotation here to enable caching. 
```suggestion
    @Cacheable(
        cacheResolver = "staticRepositoryCacheOneResolver",
        condition = "@cacheEnabledConfig.getEnabled() && #singleStudyUnfiltered"
    )
    public List<ClinicalDataBin> cachableFetchClinicalDataBinCounts(
```